src/setupProxy.js

```diff
@@ -0,0 +1,15 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const auth_endpoint = process.env.AUTH_ENDPOINT || 'http://localhost:8080';
```

Comment:
Use camelCase for variable or constant in this case

Reply:
ok, cool!